### DISCUSSION TITLE
Having trouble to start installing the daily build of Elementary OS 8

### DISCUSSION THREAD

**Discussion by marqu3s** [elementary/installer#760] [General]
Hi folks!
Anyone having trouble installing the daily build of Elementary OS 8 in a virtual machine?
I'm trying in VMWare Player and I don't even see the installer. It is just hang on a black screen.

Also created a bootable USB stick with Etcher and tried to install in a real hardware and got the same issue.

Any tips?

**Comment by danirabbit** [comment]
Hey thanks for opening a discussion, this is actually a duplicate of https://github.com/elementary/os/issues/710 :)
